### ISSUE TITLE
Error while doing batch accessibility map

### ISSUE THREAD

**Issue by greenscientist** [chairemobilite/transition#697]
We had a report of failure while doing a batch accessibility map. 

I was able to find this call stack in the logs which might be related

}
Start uploading file
Calculating accessibility map 950/5000
Error: Unable to find segment #623138866 [-73.43990526452491, 45.843344797057995] -> [-73.43913390068072, 45.84244860972186] in SweepLine tree. Please submit a bug report.
    at SweepLine.process (/app/packages/transition-common/node_modules/polygon-clipping/dist/polygon-clipping.cjs.js:1524:24)
    at Operation.run (/app/packages/transition-common/node_modules/polygon-clipping/dist/polygon-clipping.cjs.js:1761:35)
    at Object.union (/app/packages/transition-common/node_modules/polygon-clipping/dist/polygon-clipping.cjs.js:1792:20)
    at clipFunc (/app/packages/transition-common/lib/services/accessibilityMap/TransitAccessibilityMapCalculator.js:121:58)
    at Timeout._onTimeout (/app/packages/transition-common/lib/services/accessibilityMap/TransitAccessibilityMapCalculator.js:130:33)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
Error getting od trip result cannot calculate transit accessibility map with trRouting: Error: Unable to find segment #623138866 [-73.43990526452491, 45.843344797057995] -> [-73.43913390068072, 45.84244860972186] in SweepLine tree. Please submit a bug report. (TRRAM0001)
